Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -13962,6 +13967,10 @@ def main():  # pragma: no cover
         command = demisto.command()
         LOG(f'Command being called is: {command}')
 
+        # Log the API version
+        demisto.debug('Retrieving the API version')
+        demisto.debug(f'API version: {get_pan_os_version()}')
```

Comment:
Great idea. Maybe we should add this condition so as not to burden regular runs.   
```suggestion
        if is_debug_mode():
            demisto.debug('Retrieving the API version')
            demisto.debug(f'API version: {get_pan_os_version()}')
```

Reply:
Agree